src/metabase/driver/postgres.clj

```diff
@@ -370,7 +386,7 @@
       (pg-conversion identifier :numeric)
 
       (some? nfc-path)
-      (json-query identifier nfc-path)
+      (json-query identifier stored-field nfc-path)
```

Comment:
this function still only has two args...I'm assuming `stored-field` should be removed?